jupyter_notebook_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "NotebookApp": {
+    "password": "sha1:65adaa6ffb9c:36df1c2086ef294276da703667d1b8ff38f92614"
```

Comment:
Setting the password to default `sparknlp` is a great idea. I have changed the README to reflect this change.